src/main/java/randoop/generation/ForwardGenerator.java

```diff
@@ -614,6 +616,27 @@ private void randoopConsistencyTests(Sequence newSequence) {
     }
   }
 
+  /** The numeric types that "GRT Fuzzing" fuzzes. */
+  // TODO: Why are byte and char omitted?
+  // TODO: These could be tested with `==` rather than `equals()`.
```

Comment:
The only use of this Set is to check whether it is a GRT-supported type in line 836. It looks like `byte` and `char` primitive types are included? Also `equals()` behaves the same as `==` for `.class` singletons.

Reply:
Seems like this isn't a blocker